Tailviewer.AcceptanceTests/BusinessLogic/LogFiles/TextLogFileAcceptanceTest.cs

```diff
@@ -186,7 +186,7 @@ public void TestGetSection1()
 			{
 				file.Property(x => x.EndOfSourceReached).ShouldEventually().BeTrue(TimeSpan.FromSeconds(20), "because we should be able to read the entire file in a few seconds");
 				file.Count.Should().Be(165342);
-				file.GetValue(LogFileProperties.StartTimestamp).Should().Be(new DateTime(2015, 10, 7, 19, 50, 58, 982));
```

Comment:
This merge doesn't look right, GetValue(...) is the new way to retrieve data, please don't revert that.